phpBB/styles/prosilver/theme/pm.css

```diff
@@ -0,0 +1,109 @@
+/* -------------------------------------------------------------- /*
+    Styles for the refactored PM module
+/* -------------------------------------------------------------- */
+
+@media (min-width: 768px) {
+	.pm {
+		background-color: #d3d3d3;
```

Comment:
remove again look at the most recent codepen https://codepen.io/hanakin/pen/yEaJQX